package.json

```diff
@@ -40,7 +40,6 @@
     }
   },
   "volta": {
-    "node": "12.22.11",
-    "npm": "7.20.2"
+    "node": "16.14.2"
```

Comment:
**Question:** Do you know which NPM version this will use when running locally? From what I know of Volta, I think it might use the machine's NPM version as opposed to whichever NPM version usually goes what that version of Node, but I'm not sure and can't find it in the docs. ['volta which'](https://docs.volta.sh/reference/which) might be able to tell us which it will use.

Reply:
It gets the npm version that was included with the node version that you are using at that moment.
![image](https://user-images.githubusercontent.com/102036944/164228203-7ca8481f-d822-4ee5-8522-1aaaa30bd087.png)

Reply:
Nice!